torch/csrc/jit/graph_executor.cpp

```diff
@@ -364,6 +366,7 @@ struct GraphExecutorImpl {
   // in the executor. These passes go here and are always run,
   // regardless of the 'optimize' flag
   void runRequiredPasses(const std::shared_ptr<Graph>& g)  {
+    LowerLinearBlock(*g);
```

Comment:
With this and some of Richard's changes to literal handling that require us to run extra passes, it would be good to have a write up of what passes do we have, in which order are they expected to be used, and what kinds of nodes do they introduce/eliminate.

Reply:
I am going to factor out the passes in the graph executor so that there is a single file that documents this and has the code that does it. This way external clients of the derivatives can also use the passes.

Reply:
Yeah that would be a sufficient form of documentation for me. Sounds good!